tests/postgres_tests/test_array.py

```diff
@@ -363,6 +363,24 @@ class Meta:
         self.assertEqual(len(errors), 1)
         self.assertEqual(errors[0].id, 'postgres.E002')
 
+    def test_nested_field_checks(self):
+        """
+        #25867 - Nested array fields should set their leaf field's
```

Comment:
Instead of describing how things failed at one point in time, I think it's more helpful to state the expected behavior, e.g. "Nested ArrayFields are permitted." It would also be helpful to include a comment about what "postgres.E001" is.